app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009 Nikias Bassen"
+LICENSE="GNU GPL v3"
```

Comment:
Maybe we should also add:
```
	2009-2014 Martin Szulecki
	2008 Matt Colyer
```
They have each contributed almost 10% of all commits, compared to 41% for Jonathan Beck and 23% for Nikias Bassen, so maybe we should have all these 4 names in COPYRIGHT. What do you think?